experimental/Schemes/elliptic_surface.jl

```diff
@@ -2484,6 +2460,25 @@ function isomorphism_from_generic_fibers(
   A = to_FRX(a)
   B = to_FRX(b)
   img_gens = [A, B, FRX(RX[3])]
-  return morphism_from_rational_functions(X, Y, WX, WY, FRX.(img_gens); check=true)
+  m = morphism_from_rational_functions(X, Y, WX, WY, FRX.(img_gens); check=true)
```

Comment:
```suggestion
  m = morphism_from_rational_functions(X, Y, WX, WY, FRX.(img_gens); check=false)
```
I forgot this. But it should be done, eventually.